cypress/support/api.ts

```diff
@@ -1,6 +1,7 @@
-export const API_URL = 'https://bsky.social'
+export const API_URL = 'http://localhost:2583'
 export const SERVER_URL = `${API_URL}/xrpc`
 export const PLC_URL = 'https://plc.directory'
+export const HANDLE_RESOLVER_URL = 'https://bsky.social/xrpc'
```

Comment:
I recommend using the appview for handle resolution:
```suggestion
export const HANDLE_RESOLVER_URL = 'https://api.bsky.app/xrpc'
```